### PR TITLE
Add post and phone checkboxes to email-prefs

### PR DESCRIPTION
**Pull request by mariogalic**
# Test Checklist

| case                                                                                 | CODE  | PROD |
|--------------------------------------------------------------------------------------|-------|------|
| User with no consents gets default consents when repermissioning for the first time  |  ✅      | Unable to test - unable to create such a user without database hackiness that I dont want to do   |
| Existing consents are unchanged when updating new consent                            | ✅       |  ✅    |
|New default consent being added to previous set of default consents                 |  ✅      |   ✅   |
|Old consents being removed from previous consents                                            |  ✅       |  ✅       |  
|Can opt-out of phone & post, old phone and post consents are removed            |   ✅       | ✅   |  
|Select-all does not select opt-out boxes            |   ✅       |   ✅       |  
|Can set all consents with just an RP cookie            |   ✅       |    ✅      |  

## What does this change?

Add opt-outs checkboxes to `/email-prefs` for direct physical communications via post and phone.

@katyabeer

## What is the value of this and can you measure success?

GDPR

## Does this affect other platforms - Amp, Apps, etc?

Related PR: https://github.com/guardian/identity/pull/1041

## Screenshots

Email Prefs: 
screencapture-profile-thegulocal-email-prefs-2018-04-16-11_17_18

Consent Journey:
screencapture-profile-thegulocal-consents-2018-04-16-11_17_01

### CODE REVIEW

**Comment by walaura** [comment]
this line is foul looking it does not represent guardian brand values
screen shot 2018-03-26 at 10 45 34 am

File: identity/app/views/profile/privacyForm.scala.html
Comment (walaura) [inline identity/app/views/profile/privacyForm.scala.html]: i can jump in with some css for this

File: project/Dependencies.scala
Comment (walaura) [inline project/Dependencies.scala]: should we be pushing snapshot libraries?

**Review by walaura — CHANGES_REQUESTED** [review]
blocking this until the line looks nicer, but the rest is perfect!

**Review by walaura — APPROVED** [review]

File: project/Dependencies.scala
Comment (jfsoul) [reply project/Dependencies.scala]: We'll change this before it goes out 👍

**Comment by Calum-Campbell** [comment]
Waiting on approval from lawyers before merging this

File: identity/app/views/profile/privacyForm.scala.html
Comment (walaura) [inline identity/app/views/profile/privacyForm.scala.html]: we really shouldn't be duplicating this copy for compliance reasons, we should pull it from `identity`. AFAIk this is only being done because of the bold text?

**Review by walaura — COMMENTED** [review]
wanted to drop in a note about the text

File: identity/app/controllers/editprofile/ConsentsJourney.scala
Comment (jfsoul) [inline identity/app/controllers/editprofile/ConsentsJourney.scala]: Adding this to today's catch up agenda to discuss.  Doesn't block this PR because the behaviour is not changed, but I want to discuss the intended behaviour.

File: identity/app/utils/ConsentOrder.scala
Comment (jfsoul) [inline identity/app/utils/ConsentOrder.scala]: also removes old ones (which isn't obvious from the name)

**Review by jfsoul — APPROVED** [review]

**Pull request merged**
